### ISSUE TITLE
SPDY errors with Safari 8 + Piwik admin

### ISSUE THREAD

**Issue by Monarobase** [piwik/piwik#7867]
Hello,

Following up from the following forum thread:

http://forum.piwik.org/read.php?2,121502

The Piwik admin is not accessible in Safari 8+ on MacOS Yosemite when hosted with SSL on a webserver that supports SPDY.

Static files or PHP files not related to Piwik work, and when cookies are disabled there are no issues either. No other scripts that use cookies seem affected by this, only Piwik.

**Comment by mattab** [comment]
Hi @Monarobase Thanks for the report. 

If anyone else has some ideas about this bug (or if you also experience it) please leave a comment here.

**Comment by Monarobase** [comment]
Hello,

We have updated webserver, safari and piwik since I posted this bug report and my piwik admin is now accessible over spdy with safari.

I don't know what fixed it tough…

It's working now with Safari 8.0.7 on MacOS Yosemite.

**Comment by mattab** [comment]
Thanks for the update, maybe Safari fixed a bug in the browser in a recent update? Is your Piwik admin accessible over both HTTP and HTTPS?

**Comment by Monarobase** [comment]
My Piwik admin was originaly accessible with HTTP, I enabled the cookie when I was in HTTP so it wouldn't count my visits, I then later moved it to https only, so it's currently only accessible with HTTPS.

Our webserver is litespeed, which accepts both SPDY and HTTP/2 with compatible browers and HTTP 1.1 over SSL for non compatible browsers.

What was strange is this only happened with Piwik. I had this once with anouther script but after clearing my cookies it didn't happen again. However clearing cookies did non help with piwik. It's as if piwik created a cookie for the http version and Safari didn't like it over https except that even after clearing the cookie, it created a new identical one every time I viewed the Piwik admin page.
